localenv/local-dev.drawio

```diff
@@ -1 +1 @@
\ No newline at end of file
+<mxfile host="app.diagrams.net" modified="2023-03-24T12:18:56.167Z" agent="5.0 (Macintosh)" etag="NNfc1Mz1IK_Rlivegw_w" version="21.0.6" type="device"><diagram id="G7K_4BS86Fn3U5dKpeIK" name="Page-1">7VzbkuI2EP0aqjYPUL4bP85lN0nVpjI1k63MPqU0tgDtGJsYMcB+fSR8lwWYBbc8bOZlrEa+9VGfbnVLHph3882vCVrM/ogDHA4MLdgMzPuBYehjw2X/uGSbSuyxlwqmCQmyTqXgiXzHmVDLpCsS4GWtI43jkJJFXejHUYR9WpOhJInX9W6TOKzfdYGmuCF48lHYlP5NAjpLpWNbK+W/YTKd5XfWteyXOco7Z4LlDAXxuiIyPw7MuySOaXo039zhkCsv10t63qc9vxYPluCItjnh89P9cPE5eNRnj77zz/MbvsE3Qz2DZ0m3+RvjgCkga0ZxxP7dJvEqCjC/jsZacUJn8TSOUPg5jhdMqDPhN0zpNoMPrWjMRDM6D7Nf8YbQZ376yM5aX7OL8eP7TbWxrTQecELmmOIkl0XBDQeVNeM3FKaSTyTM75O+DH+DvUrKRMt4lfj4kGa8AiM2uHHMHiPZshMTHCJK3uo3QNkomxb9ilMfYsJubWiZRViGPjIrf4aTXiEzDysfPvkVKUqmmGYXKdFlB5WnKkU7zE/BfwyD/zXgVgfKAwYqo02mvVX2Dg/xkk4TRo4igkuaxK8FXRl1KBgNLXi/+WbKGXs0CeO1P0MJHQWIohe05ICvZ4TipwXaaXrN+hUYveGE4s1hlJrKz04wx3tG+7qkVjcTzSqs6mr74aop+lSt5txc0epfZIqTF4wp0+a7Vaw+bip2DKpYvaHYRxyQdzxWDYlKYceqAUTV7GGfC3/LGl9Lx82bpbfetbbV1gF/nT1fyfvaZXnfsNKOKQ936SDOQ9G8NoebKv7iDregsNxX2LAONx9PFQbzw3gVDCMS4eEahSGmQ6b4WQPPOngyeqoAdQGmMjRBU0aTqYppSZWqnM6oyv6fqg6oxzRaUlVlnqiCqhwYFLO5YX5cwXD/3PDqkd9j8kDIA2UFchSH2kjT60jqY68LLOWDzT082lQMAMVRClBaQJoW0lua/sisDZmjw6UYGX6IlkviN4Ii6YhSYv3nhlTnge/Bgl8zReMY8YPFtSlal49rnXoiyXBt0LjW1BSie4Rorw9dUwNGV2+DrpSCjR/i4LFzGgn3hGGdsUqGzR/z4NzyBfmvbPg38ISeXupjgbAc1dNL02yoDyhePR5ClObkuVV7GmlcxyeWuy5pGU5LyzibGM+D1uofkp5m15HUvJORrDCm1yvCbDssOsryGZ7gLy3YLJ8JlamCC2ycbgIb0QvowEA5x12mcldpaHUlycpwwK4SOJMjZHF6n4/zWtKf2nSMeXWrNMxuVmk4trBKA7holI+nCkvN0GKxHYZkgllEH732o2Rkm4KelJeMLKC0xDslKsttG6cpLRlZrdIP7wtFeSYLrGLQHnmlJSMLagouR14HhB6uWtQae7XhiQW0pkVau+tPwSBF6/Ixjau2YGABpWR+pDJ7fehCFwysVgmQcwsG/cfBdoWlyQbszMFqtdhGioP7Q4Ubo/CYbfKQKpyaqzagyR7zwFSuLxUaW5j1qq/QWOOG8oDCwxPy+m4tr6+NPM85bDzdVmhyEI/axdkMeB60QKtDSqbSzq+YnDQuNLFyp5nvYlx0lFOzrfryYkfc4ZC+QWee0YZdsHLaMlSw8NM+1xm2S5hCl3Xy2+/3suq9q5ArVV/UsQ2Jd3VCdtvbgLyxwyk//PDyC+tzE8xJlP/K7lbp0NAs0xGtqw+FZBqx4yR9q1uuSOKj8CaTz0kQ8LOlMNSBugAS3vEwx5HgYHaGg2wdShMHn+Pw5wIzGLQHtJ2zt11eCSKCaRiexDQMUEjA1490Tvx2J8Qvbq+yRUi6Jn67YTvNJXBLNqYRXcn2OKveZGVLti4De4E+b8+BM4+O0nJCXARuHsezDz0yDjFE6oFxyBIQFdes5X6USf9d8U+w3IZ4QstW4b1nl4uidndQ7bKdFst5YaOo5poJWRRFuoii+omI8igqx78CyYc513/5yRHtCSfsrU/Qtc/UwfMj+7XNLky+o5fdpbhuF5xdd+9m3w7se34t5p6Wqae6EHVZwpI9y7ObyjdBld+cEsvsAXE8viw5BNrvXLMTPkr5MEn7viSlPVzFXEPwyLoMKFgraTcNn8qBujhK/eAyp3cotZukY6l70W52q/Wu1KLM+pTGMiVYWaBYNb+5IcNqkruiOeJwPVUC4Z8DKVtXjlRz+i5D6hXUqnrJgFKrgg0omlsnZFiFgFbVS6SkVgWLFNRuDbD0zNn7bPdkL4VsnCnWJDtOzzjN9ILk41Bp0kBxbqaItsQddspyM067CX/A+egRTcgrGewyMNHVuvdiUUWOka3av+f29HMnl89egSZnL1c0SWD2cpsZhsYupV5wlyPSvHLuctvN+b+J3EW6IK9eRFGuMDeRcteFoijWLL+XnlpD+dV58+N/</diagram></mxfile>
```

Comment:
In the diagram, can you fix the `png` to have `(d) Rafiki Admn` -> `(d) Rafiki Admin`